tdrs-backend/tdpservice/reports/test/test_api.py

```diff
@@ -7,6 +7,27 @@
 from ..models import ReportFile
 
 
+def multi_year_report_data(user, stt):
+    """Return report data that encompasses multiple years."""
```

Comment:
I was using data that looked like this a lot for the reports tests, and it was making the code difficult to understand having it redeclared all over the place, so I pulled this utility out